modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/BaseMvPartitionStorageTest.java

```diff
@@ -47,9 +47,9 @@ public abstract class BaseMvPartitionStorageTest extends BaseMvStoragesTest {
 
     protected static final TestKey KEY = new TestKey(10, "foo");
 
-    protected static final TestValue VALUE = new TestValue(20, "bar");
+    protected static final BinaryRow BINARY_ROW = binaryRow(KEY, new TestValue(20, "bar"));
 
-    protected static final BinaryRow BINARY_ROW = binaryRow(KEY, VALUE);
+    protected static final BinaryRow BINARY_ROW2 = binaryRow(KEY, new TestValue(20, "bar"));
```

Comment:
It looks like the values of `BINARY_ROW` and `BINARY_ROW2` are identical. They should probably be different.

Reply:
Copy-paste, sorry